client/test/service.js

```diff
@@ -53,7 +53,7 @@ export const init = (
 export const handle = async (request, { store, AUTH_TOKEN }) => {
   const url = new URL(request.url)
 
-  const [_, param] = url.pathname.split('/')
+  const [_, ...pathParts] = url.pathname.split('/')
```

Comment:
Nit: I think it would be easier to read later if it was

```suggestion
  const [_, base, param, ...etc] = url.pathname.split('/')
```

Reply:
I kinda agree but I didn't do it like that because the `param` is not always the 3rd item - e.g. `/{cid}` (status - 1) and `/check/{cid}` (check - 2).